apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue

```diff
@@ -125,7 +128,7 @@ const drawChart = () => {
         const seriesColor = series.get('fill')?.toString();
         tooltip.label.adapters.add('text', (text, target) => {
             if (target?.dataItem?.dataContext) {
-                let value = target.dataItem?.dataContext?.[l.name];
+                let value = target.dataItem?.dataContext?.[legend.name];
```

Comment:
nitpick: I want to recommend you to declare it as another constant like below:

```typescript
const data = target?.dataItem?.dataContext as Data;
if(data) ...
```

I think it's more readable!